lite/examples/sound_classification/android_compose/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/compose/ui/theme/Theme.kt

```diff
@@ -0,0 +1,39 @@
+package org.tensorflow.lite.examples.soundclassifier.compose.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+private val DarkColorPalette = darkColors(
+  primary = gray900,
+  secondary = orangeLight,
+  onPrimary = Color.White,
+)
+
+private val LightColorPalette = lightColors(
+  primary = Color.White,
+  secondary = orangeDeep,
+  onPrimary = Color(0xFF425066), //gray900,
```

Comment:
Move this hard coded value to `Color.kt`

Reply:
Thanks, but I think the palette is part of the theme, and compose examples like JetNews and JetChat also define them in Themes.kt. ;)